mono/metadata/pal-icalls.h

```diff
@@ -0,0 +1,23 @@
+/**
+ * \file
+ * System.Native PAL internal calls
+ *
+ * Copyright 2018 Microsoft
+ * Licensed under the MIT license. See LICENSE file in the project root for full license information.
+*/
+
+#ifndef __MONO_METADATA_ICALL_PAL_H__
+#define __MONO_METADATA_ICALL_PAL_H__
+
+#include <metadata/object-internals.h>
+
+#if !defined(HOST_WIN32) && !defined(HOST_WASM) /* Unix System.Native PAL */
+gint32 ves_icall_CoreFX_Interop_Sys_Read (MonoSafeHandleHandle fd_handle, gchar* buffer, gint32 count);
+
+#if defined(__APPLE__)
```

Comment:
I'd move this out of the Unix like define

Reply:
No.

Reply:
I went back and forth on this and decided to keep it in, as this identifies as a Unix PAL.